test/ygrunner.py

```diff
@@ -217,7 +217,7 @@ def get_addr_and_fund(yg):
         return
     if yg.wallet_service.timelock_funded:
         return
-    addr = wallet_gettimelockaddress(yg.wallet_service.wallet, "2021-11")
+    addr = wallet_gettimelockaddress(yg.wallet_service.wallet, "2023-11")
```

Comment:
This is changed to be future date? Maybe this test could also use `@freeze_time` like in #973, so that it don't need to be changed again in future?